test/integration/RecipientTest.py

```diff
@@ -28,78 +29,100 @@ def test_search_and_pagination(self):
         self.assertIsNotNone(value)
 
     def test_lifecycle(self):
-        uuidString = str(uuid.uuid4())
-        payload = {"type": "individual", "firstName": "Tom", "lastName": "Jones",
-                   "email": "test.create" + uuidString + "@example.com"}
-        response = self.client.recipient.create(payload)
+        # Test - Create Recipient
+        response = TestHelper.createRecipient()
 
         self.assertTrue(response.name == "Tom Jones")
         self.assertTrue(response.type == "individual")
-        self.assertTrue(response.email == "test.create" +
-                        uuidString + "@example.com")
         self.assertTrue(response.lastName == "Jones")
 
         recipient_id = response.id
 
+        # Test - Update Recipient
         response = self.client.recipient.update(
             recipient_id,  {"firstName": "Jon"})
 
         self.assertTrue(response)
 
+        # Test - Delete Recipient
         response = self.client.recipient.delete(recipient_id)
         self.assertTrue(response)
 
+        # Test - Delete Multiple Recipients
+        recipient1 = TestHelper.createRecipient()
+        self.assertTrue(recipient1)
+
+        recipient2 = TestHelper.createRecipient()
+        self.assertTrue(recipient2)
+
+        payload = {
+            "ids": [
+                recipient1.id,
+                recipient2.id
+                ]
+            }
+        response = self.client.recipient.delete_multiple(payload)
+        self.assertTrue(response)
+
+        # Test - Find Recipient
```

Comment:
I guess you might want to find by  recipient1.id and recipient2.id and make sure they are all deleted?

Reply:
Sure, that's a good idea. I'll do that.